pkg/blockstorage/blockstorage_test.go

```diff
@@ -144,7 +144,7 @@ func (s *BlockStorageProviderSuite) TestCreateSnapshot(c *C) {
 }
 
 func (s *BlockStorageProviderSuite) TestSnapshotCopy(c *C) {
-	c.Skip("Sometimes, snapcopy takes over 10 minutes. go test declares failure if tests are that slow.")
+	// c.Skip("Sometimes, snapcopy takes over 10 minutes. go test declares failure if tests are that slow.")
```

Comment:
Is the intent to remove the `//` before merging to prevent checking in commented code?

Reply:
My intention of leaving this as a comment is to leave some easy breadcrumbs behind on which test took a long time in the past, in case the problem re-surface.  Alternatively, a log of test runs should also quickly provide the same information. I'll remove the `Skip` statement entirely

Reply:
Done